packages/cadl-ranch-expect/src/decorators.ts

```diff
@@ -238,6 +238,20 @@ export function $scenarioService(
     derivedModels: [],
     projectionsByName: [],
   } as any);
-  context.call($server, target, "http://localhost:3000", "TestServer endpoint");
+
+  context.call($server, target, "{endpoint}", "Test server endpoint", {
+    kind: "Model",
+    properties: new Map().set("endpoint", {
+      name: "endpoint",
+      type: context.program!.getGlobalNamespaceType().namespaces.get("TypeSpec")!.scalars.get("string"),
+      defaultValue: { valueKind: "StringValue", value: "http://localhost:3000" },
```

Comment:
`defaultValue` is a `Value` per definition: https://github.com/microsoft/typespec/blob/60d87bc77014a6756b255f3e81f931ea8b25e80f/packages/compiler/src/core/types.ts#L352

Here we use `StringValue`: https://github.com/microsoft/typespec/blob/60d87bc77014a6756b255f3e81f931ea8b25e80f/packages/compiler/src/core/types.ts#L411-L415

Reply:
This may need a `entityKind: "Value"` property.